src/test/java/io/redis/examples/QueryFtExample.java

```diff
@@ -0,0 +1,315 @@
+            System.out.println(docs1.get(i).getId());
+        }
+        // >>> bicycle:2
+        // >>> bicycle:1
+        // STEP_END
+
+        // Tests for 'ft1' step.
+        // REMOVE_START
+        Assert.assertEquals(2, res1.getTotalResults());
+        Assert.assertEquals("bicycle:2", docs1.get(0).getId());
+        Assert.assertEquals("bicycle:1", docs1.get(1).getId());
+        // REMOVE_END
+
+
+        // STEP_START ft2
+        SearchResult res2 = jedis.ftSearch("idx:bicycle","@model: ka*");
+        System.out.println(res2.getTotalResults()); // >>> 1
+
+        List<Document> docs2 = res2.getDocuments();
+
+        for (int i = 0; i < docs2.size(); i++) {
+            System.out.println(docs2.get(i).getId());
+        }
+        // >>> bicycle:4
+        // STEP_END
+
+        // Tests for 'ft2' step.
+        // REMOVE_START
+        Assert.assertEquals(1, res2.getTotalResults());
+        Assert.assertEquals("bicycle:4", docs2.get(0).getId());
+        // REMOVE_END
+
+
+        // STEP_START ft3
+        SearchResult res3 = jedis.ftSearch("idx:bicycle", "@brand: *bikes");
+        System.out.println(res3.getTotalResults()); // >>> 2
+
+        List<Document> docs3 = res3.getDocuments();
+
+        for (int i = 0; i < docs3.size(); i++) {
+            System.out.println(docs3.get(i).getId());
+        }
+        // >>> bicycle:6
+        // >>> bicycle:4
+        // STEP_END
+
+        // Tests for 'ft3' step.
+        // REMOVE_START
+        Assert.assertEquals(2, res3.getTotalResults());
+        Assert.assertEquals("bicycle:6", docs3.get(0).getId());
+        Assert.assertEquals("bicycle:4", docs3.get(1).getId());
+        // REMOVE_END
+
+
+        // STEP_START ft4
+        SearchResult res4 = jedis.ftSearch("idx:bicycle", "%optamized%");
+        System.out.println(res4.getTotalResults()); // >>> 1
+
+        List<Document> docs4 = res4.getDocuments();
+
+        for (int i = 0; i < docs4.size(); i++) {
+            System.out.println(docs4.get(i).getId());
+        }
+        // >>> bicycle:3
+        // STEP_END
+
+        // Tests for 'ft4' step.
+        // REMOVE_START
+        Assert.assertEquals(1, res4.getTotalResults());
+        Assert.assertEquals("bicycle:3", docs4.get(0).getId());
+        // REMOVE_END
+
+
+        // STEP_START ft5
+        SearchResult res5 = jedis.ftSearch(
+            "idx:bicycle",
+            new Query("%%optamised%%")
+        );
+        System.out.println(res5.getTotalResults()); // >>> 1
+
+        List<Document> docs5 = res5.getDocuments();
+
+        for (int i = 0; i < docs5.size(); i++) {
+            System.out.println(docs5.get(i).getId());
+        }
+        // >>> bicycle:3
+        // STEP_END
+
+        // Tests for 'ft5' step.
+        // REMOVE_START
+        Assert.assertEquals(1, res5.getTotalResults());
+        Assert.assertEquals("bicycle:3", docs5.get(0).getId());
+        // REMOVE_END
+
+
+// HIDE_START
+    }
```

Comment:
Closing client object at the end of application is a good practice.
```suggestion
        // REMOVE_END

// HIDE_START
        jedis.close();
    }
```

Reply:
@sazzad16 I'll also add that to the other PRs and make sure it's added to the template file too.